doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -344,8 +344,8 @@ The following strings must not be used as attribute names:
 Operations that retrieve lists of entities allow the `orderBy` URI parameter to specify the order; its 
 value can be:
 
-* The keyword "geo:proximity" to order results by distance to the center point when combined with "near" geoqueries.
-  If "geo:proximity" is used without "near" geoqueries, the order is undefined.
+* The keyword "geo:distance" to order results by distance to the center point when combined with "near" geoqueries.
```

Comment:
better 

The keyword ```geo:distance```  to order results by distance to a reference geometry when a "near" (```georel=near```) spatial relationship is used.